json-schema-converter/src/test/java/io/confluent/connect/json/JsonSchemaDataTest.java

```diff
@@ -885,6 +887,22 @@ public void testToConnectIntEnumInAllOfIsInvalid() {
     checkNonObjectConversion(expectedSchema, 1, schema, IntNode.valueOf(1));
   }
 
+  @Test
+  public void testToConnectStringConstInAllOf() {
+    JsonSchema jsonSchema = new JsonSchema("{\"const\":\"money\",\"type\":\"string\"}");
+    CombinedSchema schema = (CombinedSchema) jsonSchema.rawSchema();
+    Schema expectedSchema = new SchemaBuilder(Schema.Type.STRING).build();
+    checkNonObjectConversion(expectedSchema, "money", schema, TextNode.valueOf("money"));
+  }
+
+  @Test
+  public void testToConnectIntegerConstInAllOf() {
```

Comment:
Do we need to test with a float i.e 1.5?

Reply:
Thanks I added another test